dallinger/recruiters.py

```diff
@@ -252,9 +360,7 @@ def open_recruitment(self, n: int = 1) -> dict:
                 "prolific_estimated_completion_minutes"
             ),
             "external_study_url": self.ad_url + PROLIFIC_AD_QUERYSTRING,
-            "internal_name": "{} ({})".format(
-                self.config.get("title"), self.config.get("id")
-            ),
+            "internal_name": self.config.get("id"),
```

Comment:
Are you sure about changing this? Did you not like the previous way of doing things?

Reply:
I thought we discussed this with Nori and Frank, no? The internal name is basically used as an id.

Reply:
What I remember is agreeing not to put the id in the title. This seems done already, though. It's fine what you've proposed, don't worry, just wanted to check!

Reply:
No, we should only use the `id` because in the prolific API there is a character limit on the internal name. You can easily have longer names by taking `self.config.get("title"), self.config.get("id")` which can lead to unexpected errors. The prolific internal ID always has the same length so it would be safe to use.

Reply:
Oh cool